.github/gh_runner_dependencies.sh

```diff
@@ -25,12 +35,15 @@ echo "export GOPATH=$HOME/go" >> "$HOME"/.profile
 echo "export PATH=$PATH:/usr/local/go/bin:$GOPATH/bin" >> "$HOME"/.profile
 source "$HOME"/.profile
 
+# install the addlicense tool used to check files for license headers
 go get -u github.com/google/addlicense
 sudo ln -s "$HOME"/go/bin/addlicense /bin
 
+# install the tfenv tool to manage terraform versions
 git clone https://github.com/tfutils/tfenv.git ~/.tfenv
 sudo ln -s "$HOME"/.tfenv/bin/* /bin
 
+# install the terraform versions and configure it to the latest
 tfenv install 0.14.9
```

Comment:
nit: do we need all the `tfenv` installs? especially for the ones `>v.1.0.0`

Reply:
Fixed in https://github.com/GoogleCloudPlatform/anthos-samples/pull/218/commits/a0a7734212520f846a17c69dd54bb4beb37c673b